src/multibody/geometry.hpp

```diff
@@ -260,10 +272,11 @@ namespace se3
     /// \brief Compute the collision status between two collision objects of a given collision pair.
     ///
     /// \param[in] pair The collsion pair.
+    /// \param[out] result where the collision result is stored.
     ///
     /// \return Return true is the collision objects are colliding.
     ///
-    CollisionResult computeCollision(const CollisionPair & pair) const;
+    bool computeCollision(const CollisionPair & pair, fcl::CollisionResult& result) const;
```

Comment:
I think this is an initial design mistake. The general idea is that all computations in pinocchio needing buffer memory should use Data and GeomData for storage. Here, we are asking for an external buffer. I think we should stick to the initial desig.